src/common/components/__tests__/Suggester.test.jsx

```diff
@@ -3,10 +3,16 @@ import { shallow } from 'enzyme';
 import MockAdapter from 'axios-mock-adapter';
 
 import http from '../../http';
-import Suggester from '../Suggester';
+import Suggester, { REQUEST_DEBOUNCE_MS } from '../Suggester';
 
 const mockHttp = new MockAdapter(http);
 
+function wait(milisec = REQUEST_DEBOUNCE_MS + 1) {
+  return new Promise(resolve => {
+    setTimeout(() => resolve(), milisec);
```

Comment:
:(

Reply:
we said the ``nextTick`` would be the last one

Reply:
both are stable and non-flaky since we never have random `milisec`